frontend/app/(no-footer)/layout.tsx

```
@@ -1,8 +1,32 @@
 
 export default function RootLayout({ children }: { children: React.ReactNode }) {
-  return <>{children}</>; 
-}
\ No newline at end of file
+  return (
+    <>
+      {children}
+      
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)